src/main/java/com/sda/eventapp/mapper/EventMapper.java

```diff
@@ -55,4 +55,17 @@ public Event toEvent(EventForm form) {
                 .owner(form.getOwner())
                 .build();
     }
+
+    public List<EventView> toEventViewListClient(List<Event> events) {
```

Comment:
Is there a reason why we can't use `toEventViewList`? I'd say that REST should also return EventView with attending users, image, owner and so on. If that's not the case there should be different DTO, otherwise this method is redundant.

Method name also shouldn't be with "client" in name I think. It's too specific, it has nothing to do with client in other cases than ours. It should be more generic? Like more with REST convention naming? Not sure how to explain it, but I think the "client" suffix/prefix is unnecessary here/